Watch page layout (desktop)

matedyl opened the pull request:
https://app.clickup.com/t/2zh2dhz

Commit:
https://app.clickup.com/t/2zh2dhz

Esser50K reviewed (commented):
still have to checkout the branch and take a look

Esser50K reviewed (changes requested):
Tested it now.

I see that the video recommendations are on the right side now, but it won't collapse to a single column. I'd prefer to get this right in this PR because I'd like that every commit on Master is deployable (even when I don't deploy it)

matedyl commented on app/app/src/components/AsciiPlayer.tsx:
```diff
@@ -137,21 +137,22 @@ function AsciiPlayer(props: AsciiPlayerProps) {
-        const heightLimited = (lineLength / window.innerWidth) < (nLines / window.innerHeight)
-        const limiter = heightLimited ? window.innerHeight : window.innerWidth
+        const heightLimited = (lineLength / playerContainer.current!.clientWidth) < (nLines / window.innerHeight)
+        const fontAdjustment = 1.1  // scale adjustment for fitting the text block to the container width - depends on the width of the character in font family
```
@Esser50K I added it to fit the YT layout as closely as possible and due to the way font rendering works: it is not fully responsive and renders font in the same size for a range of `font-size` values. It has some implications on how closely the design can match YT watch page. Let's schedule a call soon and I can explain it further!

matedyl commented:
> 



> Tested it now.
> 
> I see that the video recommendations are on the right side now, but it won't collapse to a single column. I'd prefer to get this right in this PR because I'd like that every commit on Master is deployable (even when I don't deploy it)

Could you explain what you mean by collapsing to a single column? After testing I see that the main layout is a column: with video on the top and recommended vids section on the bottom.

matedyl committed:
Watch page layout on mobile (#20)

Added column layout varation to FeaturedVideos component for different layout on Watch page

Co-authored-by: mad <jetbrains@mad.solares.studio>

Esser50K committed:
adjust font adjustment to be below 1 to fit inside container
